.github/workflows/default.yml

```
@@ -357,6 +357,7 @@ jobs:
       if: always()  # We want the log even on failures.
 
   build-sbom:
+    if: ${{ github.event.pull_request.merged == true || (github.event_name == 'push' && contains(github.ref, 'refs/tags/')) }}
```

```labeled, unlabeled, opened, synchronize, reopened```
Currently, the default workflow is executed for the above types.
The pull_request.merged value exists only when it is of type 'closed'.
https://docs.github.com/ko/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-when-a-pull-request-merges

Therefore, similar to the other jobs above, it would be a good idea to execute it only when a tag is pushed.
```suggestion
    if: ${{ github.event_name == 'push' && contains(github.ref, 'refs/tags/') }}
```